package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "filevalidator",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "File signature validation in JavaScript",
   "main": "filevalidator.js",
-  "scripts": {
```

Comment:
Was this unnecessary? I don't see any tests added to the PR.

Reply:
@wojcikstefan I think his point is "why was it necessary"

Reply:
This is added by `npm init` and it's unnecessary. It was just a clean up.